src/ir/module-utils.cpp

```diff
@@ -506,9 +506,7 @@ InsertOrderedMap<HeapType, HeapTypeInfo> collectHeapTypeInfo(
   while (!newTypes.empty()) {
     while (!newTypes.empty()) {
       auto ht = newTypes.pop();
-      // TODO: Use getReferencedHeapTypes instead and remove separate
-      // consideration of supertypes below.
-      for (HeapType child : ht.getHeapTypeChildren()) {
+      for (HeapType child : ht.getReferencedHeapTypes()) {
```

Comment:
Perhaps `child` should now be renamed, as this might be the super as well?

Reply:
Eh, it's still a child in the directed graph sense.